flag.go

```diff
@@ -278,6 +278,11 @@ func (f *FlagSet) Lookup(name string) *Flag {
 	return f.lookup(f.normalizeFlagName(name))
 }
 
+// LookupByShortHand returns the Flag structure of the short handed flag, returning nil if none exists.
+func (f *FlagSet) LookupByShortHand(name byte) *Flag {
```

Comment:
I'm also unsure how I feel about `byte` vs `string`. Do people every use `byte` in their programs? Is this the best API? Making people do `var[0]` all the time? I don't love that usage inside pflag.

I'll accept your recommendation, but think about it...